webviz_subsurface/plugins/_response_parallel_coordinates.py

```diff
@@ -0,0 +1,521 @@
+                       labelStyle={"display": "inline-block"},
+                       style={"fontSize": ".80em"},
+                   )
+               ]
+            ),
+            html.Div(
+                [
+                    wcc.Select(
+                        id=self.uuid("parameter-list"),
+                        options=[
+                            {"label": ens, "value": ens} for ens in self.parameters
+                        ],
+                        multi=True,
+                        size=10,
+                        value=[],
+                        style={"marginBottom": "20px"}
+                    ),
+                ]
+            ),
+            html.Div("Filters:", style={"font-weight": "bold"}),
+            html.Div(children=self.filter_layout),]
+
+    @property
+    def layout(self):
+        """Main layout"""
+        return wcc.FlexBox(
+            id=self.uuid("layout"),
+            children=[
+                html.Div(
+                    style={"flex": 1},
+                    children=self.control_layout
+                ),
+                html.Div(
+                    style={"flex": 3},
+                    children=wcc.Graph(id=self.uuid("paralell-coords-plot")),
+                ),
+            ]
+        )
+
+    @property
+    def parallel_coords_callback_inputs(self):
+        """List of Inputs for paralell parameters callback"""
+        inputs = [
+            Input(self.uuid("exclude_include"), "value"),
+            Input(self.uuid("parameter-list"), "value"),
+            Input(self.uuid("ensemble"), "value"),
+            Input(self.uuid("responses"), "value"),
+
+        ]
+        if self.response_filters:
+            for col_name in self.response_filters:
+                inputs.append(Input(self.uuid(f"filter-{col_name}"), "value"))
+        return inputs
+
+    def make_response_filters(self, filters):
+        """Returns a list of active response filters"""
+        filteroptions = []
+        if filters:
+            for i, (col_name, col_type) in enumerate(self.response_filters.items()):
+                filteroptions.append(
+                    {"name": col_name, "type": col_type, "values": filters[i]}
+                )
+        return filteroptions
+
+    def set_callbacks(self, app):
+
+        """Set callback for paralell coordinate plot"""
+        @app.callback(
+            Output(self.uuid("paralell-coords-plot"), "figure"),
+            self.parallel_coords_callback_inputs
+        )
+        def _update_paralell_coordinate_plot(
+                exc_inc, parameter_list,
+                ensemble, response,
+                *filters):
+            """Callback to update the model for multiple regression
+            1. Filters and aggregates response dataframe per realization.
+            2. Filters parameters dataframe on selected ensemble.
+            3. Merge parameter and response dataframe.
+            5. Generate parallel parameters plot.
+            """
```

Comment:
This is not the correct docstring
```suggestion
            """Something else
            """
```